src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -92,6 +92,42 @@ def alpha_bar_fn(t):
     return torch.tensor(betas, dtype=torch.float32)
 
 
+def rescale_zero_terminal_snr(betas):
```

Comment:
is this copied from ddim? if so let's add a # Copy from statement

Reply:
It is copied from DDIM which itself is copied 1:1 (and renamed) from Page 3 Alrogithm 1 of [the ZSNR paper](https://arxiv.org/pdf/2305.08891.pdf).